docs/devctr-image.md

```diff
@@ -47,9 +47,25 @@ registry. The Firecracker CI suite must also be updated to use the new image.
 
 1. Build a new container image with the updated Dockerfile.
 
+   The below script is used to build a docker image with freezed python package
+   versions.
+
    ```bash
-    docker build -t fcuvm/dev -f tools/devctr/Dockerfile.x86_64 .
+    tools/build_docker_image.sh -f Dockerfile.x86_64
     ```
+    
+   To pull python dependencies required on the container the 
+   script uses ```requirements-freeze.txt``` which has a list of python
+   packages with their freezed versions. This ensures that same version of
+   packages are built on container in it's entire life-cycle, unless one
+   wants to intentionally update the freezed versions.  
+   #### How does requirements-freeze.txt gets updated?
+
+   After a successful docker build run, we run ```pip freeze``` on container
+   to get all the python dependencies and their respective versions.
+   We then copy the dependencies on the host to ```requirements-freeze.txt```.
+
+   If you see a diff in requirements-freeze.txt you must validate for backward compatibility and raise a PR to update the diff.  
```

Comment:
Can this ever happen without knowingly updating the `requirements-freeze` file?
Also, the line is too long, if I remember correctly, we try to keep them under 100 chars

Reply:
>Can this ever happen without knowingly updating the requirements-freeze file?
I think it can happen. a) if pip versions in Docker image gets updated, not sure tho b)new pip installs are defined in Docker images instead of defining in requirements-freeze